prow/jobs/kyma-project/template-operator/template-operator.yaml

```diff
@@ -167,4 +163,100 @@ postsubmits: # runs on main
           - name: signify-secret
             secret:
               secretName: signify-dev-secret
-  
\ No newline at end of file
+    - name: release-template-operator-build
+      annotations:
+        description: "Job to build template-operator for a release."
+        owner: "jellyfish"
+      labels:
+        prow.k8s.io/pubsub.project: "sap-kyma-prow"
+        prow.k8s.io/pubsub.runID: "release-template-operator-build"
+        prow.k8s.io/pubsub.topic: "prowjobs"
+        preset-sa-kyma-push-images: "true"
+        preset-signify-prod-secret: "true"
+      always_run: true
+      skip_report: false
+      decorate: true
+      cluster: trusted-workload
+      max_concurrency: 10
+      branches:
+        - ^v?\d+\.\d+\.\d+(?:-.*)?$
+      spec:
+        containers:
+          - image: "eu.gcr.io/sap-kyma-neighbors-dev/image-builder:v20230313-8dfce5f0b-buildkit"
+            securityContext:
+              privileged: true
+              seccompProfile:
+                type: Unconfined
+              allowPrivilegeEscalation: true
+            command:
+              - "/image-builder"
+            args:
+              - "--name=template-operator"
+              - "--config=/config/kaniko-build-config.yaml"
+              - "--context=."
+              - "--dockerfile=components/operator/Dockerfile"
+              - "--tag=$(PULL_BASE_REF)"
+            env:
+              - name: BUILDKITD_FLAGS
+                value: "--oci-worker-no-process-sandbox"
+            resources:
+              requests:
+                memory: 1.5Gi
+                cpu: 1
+            volumeMounts:
+              - name: share
+                mountPath: /home/user/.local/share/buildkit
+              - name: config
+                mountPath: /config
+                readOnly: true
+              - name: signify-secret
+                mountPath: /secret
+                readOnly: true
+        volumes:
+          - name: share
+          - name: config
+            configMap:
+              name: kaniko-build-config
+          - name: signify-secret
+            secret:
+              secretName: signify-dev-secret
+    - name: release-template-operator-assets
```

Comment:
Why is that a prowjob? Can't it be done from within the GH Action itself?

Reply:
Maybe because build pipeline has to run from certified build environment and pipeline application

Reply:
Probably if there is not image builder used it can be possible, but I don't want to investigate now how to trigger on tag creation etc. and create a GHA for it when all other module teams are releasing their module operator this way. Keep in mind that these two job are just a copy of module teams' jobs.